velox/exec/fuzzer/WriterFuzzer.cpp

```diff
@@ -162,6 +168,39 @@ class WriterFuzzer {
     return partitionCount > 0 && bucketCount > 0;
   }
 
+  // Check the table properties and see if the table is sorted.
+  bool isSorted(
+      const int32_t partitionCount,
+      const int32_t bucketCount,
+      const int32_t sortColumnCount) {
+    return partitionCount > 0 && bucketCount > 0 && sortColumnCount > 0;
```

Comment:
Shall we just check on sortColumnCount? Why we need to check partitionCount and bucketCount? We can put an assertion either in Prestissimo or Table writer that sort columns shouldn't be set if it is not partition and bucket table? thanks!